### PR TITLE
Yj

### PR DESCRIPTION
**Pull request by Royalsyj**

### CODE REVIEW

File: anyhow.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+ye anyhow
```
Comment (YoongJH) [inline anyhow.txt:1]: Wot this

File: pystonks.py [hunk 2]
```diff
@@ -23,4 +44,33 @@
 # print("Ask: " + str(NXE.info["ask"]))
 # print("Ask: " + str(GME.info["ask"]))
 # print("Ask: " + str(AMC.info["ask"]))
-# print("Ask: " + str(APHA.info["ask"]))
\ No newline at end of file
+# print("Ask: " + str(APHA.info["ask"]))
+
+
+### Added by YJ on 3 April 2021
```
Comment (YoongJH) [inline pystonks.py:50]: No need this, git commit will contain the info in git blame

**Review by YoongJH — CHANGES_REQUESTED** [review]
- [ ] remove ``` pycache pyc``` file

File: pystonks.py (hunk 2 above)
Comment (Royalsyj) [reply pystonks.py]: fien

**Pull request closed without merging**